src/app.tsx

```diff
@@ -1,48 +1,96 @@
+import { BaseLayout } from './components/layout';
 import { HashRouter as Router, Switch, Route } from 'react-router-dom';
+import { HelmetProvider, Helmet } from 'react-helmet-async';
+import { images, preloadImageList, useOnMountEffect } from './utils';
+import { FallbackRender } from '@sentry/react/dist/errorboundary';
+import { ToastContainer } from 'react-toastify';
+import { useTransactionNotifications } from './contracts';
+import * as Sentry from '@sentry/react';
 import ChainDataContextProvider from './chain-data';
-import PageContainer from './page-container';
 import Dashboard from './pages/dashboard';
-import Proposals from './pages/proposals';
-import ProposalDetailsPage from './pages/proposal-commons/proposal-details';
-import HistoryDetails from './pages/history-details';
 import History from './pages/history';
+import HistoryDetails from './pages/history-details';
+import NotFoundPage from './pages/not-found';
+import ProposalDetailsPage from './pages/proposal-commons/proposal-details';
+import Proposals from './pages/proposals';
 import Vesting from './pages/vesting';
 import './styles/variables.module.scss';
-import NotFoundPage from './pages/not-found';
 
-function App() {
+const ErrorBoundary: FallbackRender = (props) => {
+  const { error } = props;
+
+  // When the ErrorBoundary component is rendered, there must have been uncaught error somewhere in React component. We
+  // want to send this exception to Sentry.
+  useOnMountEffect(() => {
+    Sentry.captureException(error);
```

Comment:
I assume this respects if the user has opted out of Sentry?

Reply:
Unrelated to this PR, but it looks like `sendToSentry` is not being used anywhere in the app here (unless I'm missing something)

https://github.com/api3dao/api3-dao-dashboard/blob/601490afb9813f3bfd96057d1d3172f2980316a5/src/components/notifications/notifications.tsx#L91-L94